ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs

```diff
@@ -102,7 +102,7 @@ private void LoadHistogramData(AlternativeResults altResults, DamageMeasureYear
                     break;
             }
 
-            double[] binValues = histogram.BinCounts.Select(i => (double)i).ToArray();
+            double[] binValues = histogram.BinCounts.Select(binCount => (double)binCount/histogram.SampleSize).ToArray();
```

Comment:
It's not very obvious what's going on here.  The API should be doing this kind of logic, not you, why are we dividing it by the sample size?